features/FxA-107-security-event-history/README.md

```diff
@@ -0,0 +1,74 @@
+# Security Events History
+
+## Problem Summary
+
+As a Firefox Accounts user, I want to reduce the amount of sign-in
+confirmations I must make, especially when I am logging in from the same
+locations each time (my office and/or home).
+
+--------
+
+## Outcomes
+
+To start, report whenever a sign in attempt that would trigger signin
+confirmation could have been elided, based on correlating the current
+signin request with the user's history.
+
+## Metrics
+
+We can log an info message each time this would occur, tagged as
+`Account.history.seen`. A graph can be created, comparing:
+
+- Number of signin requests
+- Number of `Account.history.seen`
+
+--------
+
+## Detailed design
+
+> This is the bulk of the RFC. Explain the design in enough detail for somebody familiar
+with the language to understand.
+This should get into specifics and corner-cases, and include examples of how the feature is used.
+
+Keep a history of "security" events that occur for a user. A record in
+this history would contain these details:
+
+- `uid` - user ID from FxA
+- An event name (`account.login`, `account.reset`, `account.confirm`, etc)
```

Comment:
I think so. Imagine an attack against a large set of users whos passwords have been compromised via another breach. An attacker might try to change their passwords once they have logged in. Seeing an unusually large number of password changes should raise a red flag.